### PR TITLE
Set up continuous integration.

### PR DESCRIPTION
**Pull request by ryanchangyeolshin**

### CODE REVIEW

**Comment by ryanchangyeolshin** [comment]
image

File: index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const express = require('express')
+const createApp = require('./create-app')
+
+createApp()
+  .listen(3000 || process.env.PORT, () => console.log('Listening to PORT' + process.env.PORT))
```
Comment (thebearingedge) [inline index.js]: Just put a `PORT` in a `.env` file.

File: test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+describe('app', () => {
+
+  let _repo
+  let server
+
+  before(done => {
+    _repo = { name: 'continuous-delivery', description: 'A practice repository for testing and deployment' }
+    server = createApp()
+      .get('/', (req, res) => res.json(_repo))
```
Comment (thebearingedge) [inline test/index.test.js]: The server should already know how to respond. Don't rewrite the server code in the tests.

File: test/index.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      .listen(3000 || process.env.PORT, () => done())
```
Comment (thebearingedge) [inline test/index.test.js]: Just put a `PORT` in a `.env` file.

File: test/index.test.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  after(done => {
+    server.close(() => done())
+  })
+
+  describe('GET /', () => {
+    it('should return a json object with the repository name and description', async () => {
+      const { data } = await axios.get('http://localhost:3000/')
+      expect(data).to.deep.equal(_repo)
+      expect(data).to.not.equal('LOL')
```
Comment (thebearingedge) [inline test/index.test.js]: This assertion is not necessary.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: test/index.test.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+require('dotenv/config')
+const { describe, before, beforeEach, after, it } = require('mocha')
+const { expect } = require('chai')
+const express = require('express')
```
Comment (thebearingedge) [inline test/index.test.js]: What's this for?

**Review by thebearingedge — APPROVED** [review]
Look out for extra variables and you're good to go.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**